### ISSUE TITLE
Fix broken test 'should be protected from recursive-submission' in `test-integration-form.js`

### ISSUE THREAD

**Issue by danielrozenberg** [ampproject/amphtml#19788]
This test broke starting with Chrome 71, and was disabled in #19782. Please triage and fix

Disabled test: https://github.com/ampproject/amphtml/pull/19782/files#diff-342622682fdb4e2fbb7ff8b589e7770bR138

**Comment by aghassemi** [comment]
@cvializ let's quickly investigate to see if this is actually a code breakage issue in Chrome 71 or purely test related.

**Comment by cvializ** [comment]
sounds good, I'll take a look

**Comment by cvializ** [comment]
@aghassemi @danielrozenberg The test consistently succeeds for me in Chrome 70.0.3538.110 (long-running Chrome window FTW), Chrome 71.0.3578.80 and Chrome 73.0.3637.0, so I'm not sure what was causing it to break on Travis. Was there some condition that causes it to fail other than not being skipped? Was it non-deterministically failing?

**Comment by cvializ** [comment]
OK, I was able to make it fail consistently in Sauce Labs. The test makes a request to `localhost:38612`, which hosts the test request server in `app.js`. For some reason, this port refuses any connections in Windows + Chrome 71, but when the test runs in Windows + Chrome 70, the request returns a response and the test succeeds. I verified this with the test server running on my local machine, and Sauce Labs VMs connecting through `sc`. 

Description | Image
--|--
Chrome 70 request|<img width="674" alt="screen shot 2018-12-11 at 17 17 13" src="https://user-images.githubusercontent.com/2363700/49834053-73250700-fd69-11e8-9c04-b7a20f0ff586.png">
Chrome 70 localhost:31862|<img alt="screen shot 2018-12-11 at 17 20 38" src="https://user-images.githubusercontent.com/2363700/49833991-47098600-fd69-11e8-816b-43bee4173c5a.png">
Chrome 71 request|<img width="782" alt="screen shot 2018-12-11 at 17 19 03" src="https://user-images.githubusercontent.com/2363700/49834039-6acccc00-fd69-11e8-81cb-1498ee1fdfe7.png">
Chrome 71 localhost:31862|<img width="397" alt="screen shot 2018-12-11 at 17 19 50" src="https://user-images.githubusercontent.com/2363700/49834032-67394500-fd69-11e8-8cd4-cda96045e7ba.png">

I believe this is the root cause. Would Karma or Sauce Labs configure the server endpoint differently for different versions of Chrome?

**Comment by danielrozenberg** [comment]
/to @rsimha + @estherkim for Carlos's question

**Comment by rsimha** [comment]
Very nice sleuthing and analysis, @cvializ!

I poked around the Sauce labs documentation to see if accessing `localhost` via a port on Sauce VMs is a known problem. I found [this](https://support.saucelabs.com/hc/en-us/articles/115002212447-Unable-to-Reach-Application-on-localhost-for-Tests-Run-on-Safari-8-and-9-and-Edge) year-old article about older Safari browsers, which may offer a hint about what's going on.

There are some `port` related questions in the Sauce Connect FAQs page. [This](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy+FAQS#SauceConnectProxyFAQS-CanIAccessApplicationsonlocalhost?) one, titled "Can I Access Applications on localhost?" seemed like it might lead us to a fix.

Maybe we could try changing the port number from `38612` to one of the known proxied ports?
